config/opendatahub-io/org.yaml

```diff
@@ -177,6 +179,7 @@ orgs:
       repo-modelmesh-write:
         description: "Members with Write access to the modelmesh repository"
         members:
+        - amadhusu
```

Comment:
You don't need this access. I'm guessing you copied Humair's permissions, and his access here is an old artifact of a previous project he worked on.

Reply:
Yeah, I copied Humair's and Dharmit's permissions. I will undo these changes, thanks for the info.